src/App.vue

```diff
@@ -155,7 +160,4 @@ export default class App extends Mixins(DateMixin, FilingTemplateMixin, LegalApi
 </script>
 
 <style lang="scss" scoped>
-.app-header {
-  z-index: 1000;
```

Comment:
This was a recent change that you're undoing. Please revert this change.

Reply:
poor rebase on my part, corrected in latest commit